src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -247,79 +138,66 @@ If, however, you expect users to interact with multiple elements at the same
 time (using multi-touch), you should restrict the touch to the specific
 element.
 
-<div class="clearfix"></div>
-
-
-### TL;DR {: .hide-from-toc }
-- For full device support, handle touch, mouse and Pointer Events.
-- Always bind start event listeners to the element itself.
-- If you want the user to interact with one particular element, bind your move and end listeners to the document in the touchstart method; ensure you unbind them from the document in the end listener.
-- If you want to support multi-touch, either restrict move and end touch events to the element itself or handle all the touches on an element.
+This is more flexible for users, but does complicate the logic for manipulating
+the UI and is less resilient to user error.
 
+<div class="clearfix"></div>
 
 ### Add Event Listeners
 
-Touch events and mouse events are implemented on most mobile browsers.
+In Chrome (version 55+) and Internet Explorer + Edge `PointerEvents` are
+the recommended approach for implementing custom gestures.
+
+In other browsers `TouchEvents` and `MouseEvents` are the approach to take..
 
-The event names you need to implement are `touchstart`, `touchmove`,
-`touchend` and `touchcancel`.
+The great feature of `PointerEvents` is that it merges multiple types of input,
+including mouse, touch and pen events, into one set of
+callbacks. The events to list for are `pointerdown`, `pointermove` and
+`pointerup`.
```

Comment:
If you're going to mention `touchcancel` below then you should probably also mention `pointercancel` which serves a similar purpose (both are pretty obscure these days - eg. if the tab changes while a finger is on the screen).